### ISSUE TITLE
Sankaku complex grabber crash when you click on a image

### ISSUE THREAD

**Issue by MrAndre96** [Bionus/imgbrd-grabber#1364]
**Bug description**

When you click on a image with the source sankaku complex the program will crash.

**Steps to reproduce**

Just open a image for example when you search nekopara or something different and the program will not respond untill it will crash at the end. (Have also deleted my settings and it did the same thing)

**Expected behavior**

Like with the other sources tou aspect that it will not crash when you click on a image

**System information**

Windows 10 with the nightly version

**Comment by Bionus** [comment]
Indeed. More than a crash, it causes an infinite loop, which ends up crashing.
